mapstory/static/style/site/base.less

```diff
@@ -65,7 +66,7 @@ select {
 
 p, ul {
   font-size: 18px;
-  font-weight: 100;
+  font-weight: 200;
```

Comment:
The switch to consistently using Lato (to match Composer designs I've been working on) meant that `font-weight: 100` wasn't super legible, so I made these changes. At some point in the future I'd like to go through `master` and take font weight settings here out all together, because we're having to UNSET font-weight or override it all over the CSS, which is a code smell that indicates we shouldn't be adding those properties to base elements like we do here.

That said, that was a deep rabbit hole that I started to go down before I realized that separating concerns with this PR was probably a good idea ;) That's a project for another day...